packages/playground/src/components/node_selector/TfDomainName.vue

```diff
@@ -147,7 +147,6 @@ export default {
       size: window.env.PAGE_SIZE,
       page: Math.max(1, pagination.value.page),
       farmId: enableCustomDomain.value ? props.farm?.farmId : undefined,
-      availableFor: gridStore.client.twinId,
```

Comment:
why did you remove this ?

Reply:
It's seeking for the twinId when you logout from the dashboard when you are already on the domains page and the page blocks on that error so you have to refresh the page to apply the logout method.

Reply:
but i think this will break the filter logic!, available for is needed while listing the nodes in the domains

Reply:
maybe it needs to be optional to avoid breaking on logout, let me check